tshirts.c

```diff
@@ -1,22 +1,24 @@
-#include <stdio.h>
-#include <assert.h>
-
+#include "tshirt.h"
 char size(int cms) {
     char sizeName = '\0';
-    if(cms < 38) {
+    if(cms < TSHIRTSIZE_S) {
         sizeName = 'S';
-    } else if(cms > 38 && cms < 42) {
+    } else if(cms > TSHIRTSIZE_S && cms < TSHIRTSIZE_L) {
         sizeName = 'M';
-    } else if(cms > 42) {
+    } else if(cms > TSHIRTSIZE_L) {
         sizeName = 'L';
     }
     return sizeName;
 }
 
+void testTShirt()
+{
+    assert(size(TSHIRTSIZE_S) == 'S');

```

Comment:
All boundary values and the values in between need to be tested, i.e. 37, 38, 40, 42, 43 etc.